doc/coq.rst

```diff
@@ -0,0 +1,494 @@
+  way that OCaml ``wrapped`` libraries do. For compatibility reasons, the Coq
+  v1.0 installs a theory named ``foo.Bar`` under ``foo/Bar``. Also note that Coq
+  supports composing a module path from different theories, thus you can name a
+  theory ``foo.Bar`` and a second one ``foo.Baz``, and things will work
+  properly.
+
+- The ``modules`` field enables constraining the set of modules included in the
+  theory, similarly to its OCaml counterpart. Modules are specified in Coq
+  notation. That is to say, ``A/b.v`` is written ``A.b`` in this field.
+
+- If ``package`` is present, Dune will generate install rules for the ``.vo``
+  files on the theory. ``pkg_name`` must be a valid package name. Note that Coq
+  v1.0 uses the Coq legacy install setup, where all packages share a common root
+  namespace and install directory, ``lib/coq/user-contrib/<module_prefix>``, as
+  customary in the Make-based Coq package ecosystem. For compatibility, we also
+  install under the ``user-contrib`` prefix the ``.cmxs`` files that appear in
+  ``<ocaml_libraries>``,
+
+- ``<coq_flags>`` will be passed to ``coqc`` as command-line options.
+  ``:standard`` is taken from the value set in the ``(coq (flags <flags>))``
+  field in ``env`` profile. See :ref:`dune-env` for more information.
+
+- The path to the ``<ocaml_libraries>`` installed locations will be passed to
+  ``coqdep`` and ``coqc`` using Coq's ``-I`` flag; this allows for a Coq theory
+  to depend on a ML plugin,
+
+- Your Coq theory can depend on other theories by specifying them in the
+  ``<coq_theories>`` field. Dune will then pass to Coq the corresponding flags
+  for everything to compile correctly [ ``-Q`` ]. As of today, we only support
+  composition with libraries defined in the same scope (that is to say, under
+  the same ``dune-project`` domain). We will lift this restriction in the
+  future. Note that composition with the Coq's standard library is supported,
+  but in this case the ``Coq`` prefix will be made available in a qualified way,
+  since Coq lang 0.2.
+
+- You can enable the production of Coq's native compiler object files by setting
+  ``<coq_native_mode>`` to ``native``. This will pass ``-native-compiler on`` to
+  Coq and install the corresponding object files under ``.coq-native``, when in
+  ``release`` profile. The regular ``dev`` profile will skip native compilation
+  to make the build faster, available since Coq v0.3.
+
+  Please note: support for native compute is **experimental**, and requires Coq
+  >= 8.12.1; moreover, depending libraries *must* be built with ``(mode
+  native)`` for this to work. Also, Coq must be configured to support native
+  compilation. Note that Dune will explicitly disable output of native
+  compilation objects when ``(mode vo)``, even if the default Coq's configure
+  flag enabled it. This will be improved in the future.
+
+Coq Documenation
+~~~~~~~~~~~~~~~~
+
+Given a :ref:`coq-theory` stanza with ``name A``, dune will produce two
```

Comment:
```suggestion
Given a :ref:`coq-theory` stanza with ``name A``, Dune will produce two
```

Reply:
This confuses me, is it `dune` or `Dune` as a name?

Reply:
It is confusing! When I went through all the Dune files a few months back, we had a discussion about this. Dune, capitalised like this, is the proper name of the build system. When talking about it as a whole, it's written Dune. When talking about the command or the `dune` file, it's wrapped in monospace like that. This gives a visual differentiation between them.